ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml

```diff
@@ -49,6 +64,31 @@ tests:
     make test
   container:
     from: src
+- as: e2e-aws-operator
+  steps:
+    cluster_profile: aws
```

Comment:
I would recommend that you extract these steps into a workflow in the step registry. But it's not a hard requirement for this PR.

Reply:
Workflows seem to be more of a reusable component, since the additional step we added is too specific, I'm inclined to leave it as is for now.